packages/transform-github-emoji/src/index.js

```diff
@@ -4,7 +4,10 @@ const { emojify } = require('node-emoji');
 
 /**
  * Transform a string using node-emoji.
- *
+ * @example
+ * <!--emdaer-t
+ *   - '@emdaer/transform-github-emoji'
+ * -->
```

Comment:
This is fixed. It was because the example was inlining the transform call before the one at the bottom of the time. Therefore, it was finding the example first and blanking it out. I fixed this by removing the call above the example. I did this in the other transforms too.